src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/jpa/repository/VisitIdRepository.kt

```diff
@@ -0,0 +1,11 @@
+package uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.jpa.repository
+
+import org.springframework.data.repository.CrudRepository
+import org.springframework.stereotype.Repository
+import uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.jpa.VisitId
+import java.util.Optional
+
+@Repository
+interface VisitIdRepository : CrudRepository<VisitId, Long> {
+  fun findOneByVsipId(vsipId: String): Optional<VisitId>
```

Comment:
don't need to use `Optional` in kotlin, should be able to use `?` instead.  See https://docs.spring.io/spring-data/jpa/docs/2.3.0.RELEASE/reference/html/#repositories.nullability.kotlin

Reply:
done